src/App.vue

```diff
@@ -1,26 +1,23 @@
 <template>
-  <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png" />
-    <HelloWorld msg="Hello Vue 2 + Vite" />
+  <div>
+    <NavBar />
+    <main id="app">
+      <!-- <img alt="Vue logo" src="./assets/logo.png" /> -->
+      <router-view></router-view>
+    </main>
   </div>
 </template>
 
 <script>
-import HelloWorld from "./components/HelloWorld.vue";
+import NavBar from "@/components/layout/NavBar.vue";
```

Comment:
```suggestion
import NavBar from "@/components/layout/NavBar";
```

Reply:
It's not a blocker, but when you import components in the .vue file you can skip the extension in the components import

Reply:
For some reason It didn't work by default for me so I had to update my `vite.config.ts` by adding `extensions` prop to it.

Reply:
Not a blocker. Looks like it is how vite works :)